java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/order/OrderController.java

```diff
@@ -1,59 +1,36 @@
 package com.my.FoodTruckApp.order;
 
-import com.my.FoodTruckApp.appetizer.AppetizerService;
-import com.my.FoodTruckApp.entree.EntreeService;
 import lombok.RequiredArgsConstructor;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.*;
-import org.springframework.web.server.ResponseStatusException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
 
 import java.util.ArrayList;
 
-@RestController //this is our API call; recieving the API call is the main job of the controller; it creates endpoints with URLs
-@RequiredArgsConstructor //looks for all fields but only uses FINAL fields; this is for the dependency injection
+@RestController
+@RequiredArgsConstructor
+@Slf4j
 public class OrderController {
 
-    private final EntreeService entreeService; //getting an instance of the service, THIS IS THE DEPENDENCY INJECTION
-    private final AppetizerService appetizerService; //spring will look through the variables and see if they have a DEPENDENCY on this service
     private final OrderService orderService;
 
-    @GetMapping("/order")
-    public String gettingOrder() {
-        return "Hello from order controller";
-        //must make appetizer and entrees viewable and usable by this file
-    }
-    //------------------------get list of orders-----------------------------------------------------------------------------------------------------------
     @GetMapping("/orders")
-    public ResponseEntity<ArrayList<Order>> getListOfOrders() {
-        ArrayList<Order> orders = orderService.getListOfOrders();
-        return new ResponseEntity<>(orders, HttpStatus.OK);
+    public ArrayList<Order> getListOfOrders() {
+        return orderService.getListOfOrders();
     }
+
     //------------------------get order by ID-----------------------------------------------------------------------------------------------------------
-    @GetMapping("/orders/{id}")
-    public Order getOrderById(@PathVariable Integer id) {
-        return orderService.getOrderById(id).orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND));
-    }
-    //------------------------create order-----------------------------------------------------------------------------------------------------------
+//    @GetMapping("/orders/{id}")
+//    public Order getOrderById(@PathVariable Integer id) {
+//        return orderService.getOrderById(id).orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND));
+//    }
     @PostMapping("/orders")
```

Comment:
do you need this? If so, can this be official documentation?

Reply:
I do not, it has been removed